start_time_intent_handler.py

```diff
@@ -151,20 +161,23 @@ def _ComputePrayerTimeAndRespond(self, desired_prayer, lat, lng, city):
     return self._MakeSpeechResponse(canonical_prayer, desired_prayer, prayer_time, city)
 
 
-  def _MakeSpeechResponse(self, canonical_prayer, desired_prayer, prayer_time, city):
```

Comment:
it's technically not correct to say "the time for zuhr is 5 PM in MCA" - it should be saying "at MCA". we can fix this separately - probably makes sense to put it in response_builder when we do clean it up.